wetterdienst/radolan.py

```diff
@@ -0,0 +1,67 @@
+""" Functions for DWD RADOLAN collection """
+from pathlib import Path
+from typing import Union
+import datetime as dt
+
+import pandas as pd
+
+from wetterdienst.constants.access_credentials import DWDCDCDataPath
+from wetterdienst.constants.metadata import DWD_FOLDER_MAIN
+from wetterdienst.data_storing import store_radolan_data, restore_radolan_data
+from wetterdienst.download.download import download_radolan_data
+from wetterdienst.enumerations.column_names_enumeration import DWDMetaColumns
+from wetterdienst.enumerations.parameter_enumeration import Parameter
+from wetterdienst.enumerations.time_resolution_enumeration import TimeResolution
+from wetterdienst.enumerations.period_type_enumeration import PeriodType
+from wetterdienst.exceptions.start_date_end_date_exception import DatetimeOutOfRangeError
+from wetterdienst.indexing.file_index_creation import create_file_index_for_radolan
+
+
+def collect_radolan_data(
+        datetime: Union[str, dt.datetime],
+        time_resolution: TimeResolution,
+        folder: Union[str, Path] = DWD_FOLDER_MAIN,
+):
+    datetime = pd.to_datetime(datetime).replace(tzinfo=None)
+
+    # RADOLAN is only offered ina 10 minute interval respective 1hour
```

Comment:
Pandas DateTimeIndex offers a round function. 
```
pd.date_range(datetime(2020, 2, 1), datetime(2020, 2, 1, 12), freq='3min').round('10min')
```
And drop duplicates ;)

Reply:
Good point and saves some error handling!